app/ui.js

```diff
@@ -1099,14 +1096,49 @@ var UI;
             // Don't display the connection settings until we're actually disconnected
         },
 
+        reconnect: function() {
+            delete UI.reconnect_callback;
+
+            // if reconnect has been disabled in the meantime, do nothing.
+            if (UI.inhibit_reconnect) {
+                return;
+            }
+
+            UI.connect();
+        },
+
         disconnectFinished: function (rfb, reason) {
+            var reconnect = UI.getSetting('reconnect');
+
+            if ((reconnect === 'true' || reconnect == '1') && !UI.inhibit_reconnect) {
+                document.getElementById("noVNC_transition_text").innerHTML = _("Reconnecting...");
```

Comment:
I'll do that. Please note that the other two transition text modifications also use `innerHTML`:
```javascript
case 'connecting':
    document.getElementById("noVNC_transition_text").innerHTML = _("Connecting...");
    document.documentElement.classList.add("noVNC_connecting");
    break;
[...]
case 'disconnecting':
    UI.connected = false;
    document.getElementById("noVNC_transition_text").innerHTML = _("Disconnecting...");
    document.documentElement.classList.add("noVNC_disconnecting");
    break;
```
I was unable to find the fix, is it maybe a good idea to also apply the fix in these two places?